### PR TITLE
Fix linters

### PR DESCRIPTION
**Pull request by RMadjev**

### CODE REVIEW

**Review by atodorov — CHANGES_REQUESTED** [review]
CI jobs are failing with what seems to be a dependency issue. 

It looks like we either need to pin some package versions or move to PHP 7.4 but I can't say what the implications of any of this is.  What is your proposed solution ?

**Comment by atodorov** [comment]
I see a failure in CI complaining about password not being set and I see that .travis-ci.yml calls phpunit but I don't see executing the linters anywhere. Where are they executed from ?

File: .travis.yml [hunk 1]
```diff
@@ -2,13 +2,15 @@ language: php
...
 install:
-  - composer install
+  - composer self-update --rollback && composer install
 
 script:
-- ./vendor/bin/phpunit
+- export TCMS_PRODUCT=$TRAVIS_REPO_SLUG
+- export TCMS_PRODUCT_VERSION=$TRAVIS_PULL_REQUEST_BRANCH
+- export TCMS_BUILD=$TRAVIS_BUILD_NUMBER
+- ./vendor/bin/phpunit tests/
```
Comment (atodorov) [inline .travis.yml:10]: This is missing the configuration for `~/.tcms.conf`, see https://github.com/kiwitcms/tap-plugin/blob/master/.github/workflows/testing.yml#L33

The Travis CI environment defines the following variables: TCMS_API_URL,  TCMS_PASSWORD , TCMS_USERNAME, TCMS_VERIFY_SSL_CERTIFICATES

**Review by atodorov — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (RMadjev) [reply .travis.yml:10]: We dont need the tcms.conf, when its missing the plugin will try to read the environment variables. I will remove the TCMS_PRODUCT/BUILD/VERSION as the plugin try to find the TRAVIS variables by itself. At the moment it cant find the PASSWORD variable, I will investigate it.

Comment (atodorov) [reply .travis.yml:10]: Protected variables are not exposed to PRs from forks. Open this PR from a branch on the main repository.

**Pull request closed without merging**